src/EA.Weee.Web/Areas/AATFReturn/Mappings/ToViewModel/ReturnToReturnViewModelMap.cs

```diff
@@ -29,7 +29,7 @@ public ReturnViewModel Map(ReturnData source)
                 }
             }
 
-            return new ReturnViewModel(source.Quarter, source.QuarterWindow, source.Quarter.Year, CheckIfTonnageIsNull(NonObligatedTonnageTotal), CheckIfTonnageIsNull(NonObligatedTonnageTotalDcf), source.ReturnOperatorData);
+            return new ReturnViewModel(source.Quarter, source.QuarterWindow, source.Quarter.Year, CheckIfTonnageIsNull(NonObligatedTonnageTotal), CheckIfTonnageIsNull(NonObligatedTonnageTotalDcf));
```

Comment:
Use the ToTonnageDisplay extension method

Reply:
Updated